src/main/java/org/mockito/MockitoFramework.java

```diff
@@ -42,7 +42,7 @@
      * @since 2.1.0
      */
     @Incubating
-    MockitoFramework addListener(MockitoListener listener);
```

Comment:
I reviewed the implementation and it seems like an overkill. What's the goal? See my comment to the entire PR.